### DISCUSSION TITLE
Spacy model to tensorflow model

### DISCUSSION THREAD

**Discussion by ande4485** [explosion/spaCy#12546] [Help: Coding & Implementations] [answered]
Hi,

I created a spacy model with ner and text classification. This model works great but I'm wondering if I can export it to tensorflow Lite to test it on a mobile device. Do you think it's possible ? I search with Thinc, TensorflowWrapper etc ... but I didn't understand if it a solution. I didn't find an example for Tensorflow ( only for pytorch).

Many thanks

**Comment by svlandeg** [comment, marked as answer]
Hi! The spaCy models you'll have trained are in fact Thinc [models](https://thinc.ai/docs/api-model), and currently there is no functionality available to export those directly to Tensorflow Lite.

What the `TensorflowWrapper` is supposed to do, is allow you to specify your model in Tensorflow, than load it - through the wrapper - as Thinc model to be used in spaCy. But that's kind of the reverse of what you're looking for :/

**Reply by ande4485** [reply]
Thank you @svlandeg , but is there an example with tensorflow ? Because there's just an example with Torch.

**Reply by svlandeg** [reply]
Examples to use the `TensorflowWrapper` can be found in our test suite: [here](https://github.com/explosion/thinc/blob/master/thinc/tests/layers/test_mnist.py#L47) and [here](https://github.com/explosion/thinc/blob/master/thinc/tests/layers/test_tensorflow_wrapper.py), further examples in Thinc's docs [here](https://thinc.ai/docs/usage-frameworks#integrating-models), and the API specification of the wrapper is [here](https://thinc.ai/docs/api-layers#tensorflowwrapper).
